website/client/src/components/tasks/spells.vue

```diff
@@ -34,34 +26,55 @@
       >
         <div slot="drawer-slider">
           <div class="container spell-container">
-            <div class="row">
+            <div class="spell-container-row">
               <!-- eslint-disable vue/no-use-v-if-with-v-for -->
               <div
                 v-for="(skill, key) in spells[user.stats.class]"
                 v-if="user.stats.lvl >= skill.lvl"
```

Comment:
This should be removed because it's causing skills you don't have access to yet to be hidden completely while they should use the new disabled state
<img width="1123" alt="Schermata 2020-11-28 alle 19 40 19" src="https://user-images.githubusercontent.com/589911/100523592-ef60c100-31b1-11eb-81ca-34403b9dd861.png">
<img width="221" alt="Schermata 2020-11-28 alle 19 42 57" src="https://user-images.githubusercontent.com/589911/100523593-f1c31b00-31b1-11eb-8799-e4fc504fd1b1.png">